WebViewControl/HttpResourceHandler.cs

```diff
@@ -26,7 +28,11 @@ internal class HttpResourceHandler : DefaultResourceHandler {
                     var response = await httpRequest.GetResponseAsync();
                     Response = response.GetResponseStream();
                     Headers = response.Headers;
-                    Headers.Add("Access-Control-Allow-Origin", "*"); // we have to smash any existing value here
+
+                    // we have to smash any existing value here
+                    Headers.Remove(AccessControlAllowOriginHeaderKey);
+                    Headers.Add(AccessControlAllowOriginHeaderKey, "*"); 
+
```

Comment:
catch {
    // we should catch exceptions.. network errors cannot crash the app
}